src/components/focus_trap/focus_trap.tsx

```diff
@@ -16,6 +16,10 @@ import { findElementBySelectorOrRef, ElementTarget } from '../../services';
 export type FocusTarget = ElementTarget;
 
 interface EuiFocusTrapInterface {
+  /**
+   * Whether `onClickOutside` should be called on mouseup instead of mousedown
```

Comment:
```suggestion
   * Whether `onClickOutside` should be called on mouseup instead of mousedown.
   * This flag can be used to prevent conflicts with outside toggle buttons by delaying the closing click callback.
```

More prop documentation suggestions, I always find it useful to know why/when to use certain props and not just what it does